src/main/java/taxi/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi?serverTimezone=UTC";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "87654321*aB";
```

Comment:
looks like the password you use. if so, it is better to delete it)

Reply:
it's OK, it's just a password like "12345678" :)